src/code/z_demo.c

```diff
@@ -391,176 +443,189 @@ void func_80064824(PlayState* play, CutsceneContext* csCtx, CsCmdBase* cmd) {
+        play->envCtx.lightSettingOverride = cmd->settingPlusOne - 1;
         play->envCtx.lightBlend = 1.0f;
     }
 }
 
-// Command 0x56: Play Background Music
-void Cutscene_Command_PlayBGM(PlayState* play, CutsceneContext* csCtx, CsCmdMusicChange* cmd) {
-    if (csCtx->frames == cmd->startFrame) {
-        func_800F595C(cmd->sequence - 1);
+void CutsceneCmd_StartSequence(PlayState* play, CutsceneContext* csCtx, CsCmdStartSeq* cmd) {
+    if (csCtx->curFrame == cmd->frame) {
+        func_800F595C(cmd->seqIdPlusOne - 1);
     }
 }
 
-// Command 0x57: Stop Background Music
-void Cutscene_Command_StopBGM(PlayState* play, CutsceneContext* csCtx, CsCmdMusicChange* cmd) {
-    if (csCtx->frames == cmd->startFrame) {
-        func_800F59E8(cmd->sequence - 1);
+void CutsceneCmd_StopSequence(PlayState* play, CutsceneContext* csCtx, CsCmdStopSeq* cmd) {
+    if (csCtx->curFrame == cmd->frame) {
+        func_800F59E8(cmd->seqIdPlusOne - 1);
     }
 }
 
-// Command 0x7C: Fade Background Music over duration
-void Cutscene_Command_FadeBGM(PlayState* play, CutsceneContext* csCtx, CsCmdMusicFade* cmd) {
-    u8 var1;
+void CutsceneCmd_FadeSequence(PlayState* play, CutsceneContext* csCtx, CsCmdFadeSeq* cmd) {
+    u8 duration;
 
-    if ((csCtx->frames == cmd->startFrame) && (csCtx->frames < cmd->endFrame)) {
-        var1 = cmd->endFrame - cmd->startFrame;
+    if ((csCtx->curFrame == cmd->startFrame) && (csCtx->curFrame < cmd->endFrame)) {
+        duration = cmd->endFrame - cmd->startFrame;
 
-        if (cmd->type == 3) {
-            Audio_QueueSeqCmd(var1 << 0x10 | (0x1 << 28 | SEQ_PLAYER_FANFARE << 24 | 0xFF));
+        if (cmd->type == CS_FADE_FANFARE) {
+            Audio_QueueSeqCmd(duration << 0x10 | (0x1 << 28 | SEQ_PLAYER_FANFARE << 24 | 0xFF));
         } else {
-            Audio_QueueSeqCmd(var1 << 0x10 | (0x1 << 28 | SEQ_PLAYER_BGM_MAIN << 24 | 0xFF));
+            Audio_QueueSeqCmd(duration << 0x10 | (0x1 << 28 | SEQ_PLAYER_BGM_MAIN << 24 | 0xFF));
         }
     }
 }
 
-// Command 9: ?
-void Cutscene_Command_09(PlayState* play, CutsceneContext* csCtx, CsCmdUnknown9* cmd) {
-    if (csCtx->frames == cmd->startFrame) {
+void CutsceneCmd_RumbleController(PlayState* play, CutsceneContext* csCtx, CsCmdRumble* cmd) {
+    if (csCtx->curFrame == cmd->startFrame) {
         func_800AA000(0.0f, cmd->unk_06, cmd->unk_07, cmd->unk_08);
     }
 }
 
-// Command 0x8C: Set Time of Day & Environment Time
-void func_80065134(PlayState* play, CutsceneContext* csCtx, CsCmdDayTime* cmd) {
-    s16 temp1;
-    s16 temp2;
+void CutsceneCmd_SetTime(PlayState* play, CutsceneContext* csCtx, CsCmdTime* cmd) {
+    s16 hours;
+    s16 minutes;
 
-    if (csCtx->frames == cmd->startFrame) {
-        temp1 = (cmd->hour * 60.0f) / (360.0f / 0x4000);
-        temp2 = (cmd->minute + 1) / (360.0f / 0x4000);
+    if (csCtx->curFrame == cmd->frame) {
+        hours = (cmd->hour * 60.0f) / (360.0f / 0x4000);
+        minutes = (cmd->minute + 1) / (360.0f / 0x4000);
```

Comment:
```suggestion
        hours = (cmd->hour * 60.0f) / (24.0f * 60.0f / 0x10000);
        minutes = (cmd->minute + 1) / (24.0f * 60.0f / 0x10000);
```
Feel like this constant expression is clearer to what it is. In MM we even have a couple macros for this conversion.